src/backend/looped/self/super_jump.cpp

```diff
@@ -0,0 +1,31 @@
+#include "natives.hpp"
+#include "backend/looped_command.hpp"
+#include "gta/enums.hpp"
+
+namespace big
+{
+
+	class super_jump_looped : looped_command
+	{
+		using looped_command::looped_command;
+
+		virtual void on_tick() override
+		{
+			if (PAD::IS_CONTROL_JUST_PRESSED(0, (int)ControllerInputs::INPUT_JUMP) && !PED::IS_PED_IN_ANY_VEHICLE(self::ped, NULL) && !ENTITY::IS_ENTITY_IN_AIR(self::ped))
+			{
+				ENTITY::APPLY_FORCE_TO_ENTITY_CENTER_OF_MASS(self::ped, 0, 0, 10000, 5000, true, true, true, false);
+			}
+			if (PED::IS_PED_FALLING(self::ped))
```

Comment:
If your worried about ragdolling you can use ``SET_PED_CAN_RAGDOLL`` & ``SET_PED_CAN_RAGDOLL_FROM_PLAYER_IMPACT``.

Reply:
I will see if that has the same effect as my version when i get out of school. Ill check it out in less than 2 hours

Reply:
nope, the reason i have clear tasks is so the following animation doesn't occur: 

https://user-images.githubusercontent.com/106003542/211103428-b3bcf3f0-6db6-49aa-b70a-b1d3719ea2d5.mp4